packages/teams-js/src/internal/interfaces.ts

```diff
@@ -50,3 +50,15 @@ export interface DOMMessageEvent {
   func: string;
   args?: any[];
 }
+
+/**
+ * @hidden
+ * Meant for providing information related to certain callback context.
```

Comment:
Will this be only used in telemetry? If so, might use a name that reflecting it is telemetry related?

CallbackInformation sounds a bit generic.

Reply:
Yeah, I agree that is best to separate it to its own telemetry naming convention. Will correct it.